### ISSUE TITLE
contribute Mobile Development Projects

### ISSUE THREAD

**Issue by AkashSingh3031** [AkashSingh3031/The-Complete-FAANG-Preparation#279]
👋 Hello everyone,
You are all welcome to contribute to **[Mobile Development](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation/tree/master/4%5D.%20Projects/Mobile%20Development)**

**Comment by geeky8** [comment]
Interested - flutter related work

**Comment by AkashSingh3031** [comment]
> Interested - flutter related work

You can contribute your work in `mobile development/Flutter` folder

**Comment by SuriyaaVijay** [comment]
Can I contribute a Android Studio Project written in java

**Comment by AkashSingh3031** [comment]
> Can I contribute a Android Studio Project written in java

Yes, you can contribute your project but if you write in java then 1st create java folder then add project in that folder

**Comment by NaveenGali11** [comment]
Hey, Akash, can i contribute React Native Project to the mobile development projects.

**Comment by AkashSingh3031** [comment]
> 
> Hey, Akash, can i contribute React Native Project to the mobile development projects.

Yes, you can contribute
But also create `React Native` folder inside `Mobile Development` folder and also add demo video of your project in `Readme.md` file in same folder

**Comment by aanujkhurana** [comment]
Hi Akash, I was hoping to contribute two simple iOS Swift apps to Mobile Development
1. Checklist App
2. Saving Location using map and coordinates App

**Comment by AkashSingh3031** [comment]
> Hi Akash, I was hoping to contribute two simple iOS Swift apps to Mobile Development
> 1. Checklist App
> 2. Saving Location using map and coordinates App

You are welcome to contribute
But add your project in iOS Application
And also add demo video in readme file in same folder
